packages/boxed-expression-component/src/contextMenu/MenuWithHelp/MenuWithHelp.tsx

```diff
@@ -0,0 +1,20 @@
+import { Menu, MenuGroup, MenuList } from "@patternfly/react-core/dist/js/components/Menu";
+import * as React from "react";
+import "./MenuWithHelp.css";
+
+export interface MenuWithHelpProps {
+  children: React.ReactElement;
+  menuGroupLabel?: string;
+  onMenuItemSelected: (event?: React.MouseEvent, itemId?: string | number) => void;
+  selectedMenuItem?: any;
+}
+
+export function MenuWithHelp({ children, menuGroupLabel, onMenuItemSelected, selectedMenuItem }: MenuWithHelpProps) {
+  return (
+    <Menu onSelect={onMenuItemSelected} selected={selectedMenuItem}>
+      <MenuGroup className="menu-with-help" label={menuGroupLabel ?? ""}>
+        <MenuList>{children}</MenuList>
+      </MenuGroup>
+    </Menu>
+  );
+}
```

Comment:
Does this need to exist? I mean, the only thing this is adding to the <Menu> component is the `menu-with-help` CSS class... I don't think that justifies the existance of this component, to be honest. I think we'll be better with the MenuItemWithHelp.tsx alone.

Reply:
removing `MenuWithHelp`